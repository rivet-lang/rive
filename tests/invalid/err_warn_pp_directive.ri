#error a simple error
#warn a simple warn

#if true
    #warn if true executed
#else
    #error if else executed
#endif