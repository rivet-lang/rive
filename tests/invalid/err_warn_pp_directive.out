tests/invalid/err_warn_pp_directive.ri:1:8: error: #error: a simple error
    1 | #error a simple error
      |        ^~~~~~~~~~~~~~
tests/invalid/err_warn_pp_directive.ri:2:7: warning: #warn: a simple warn
    2 | #warn a simple warn
      |       ^~~~~~~~~~~~~
tests/invalid/err_warn_pp_directive.ri:5:11: warning: #warn: if true executed
    5 |     #warn if true executed
      |           ^~~~~~~~~~~~~~~~
rivet: error: could not compile module `err_warn_pp_directive`, aborting due to previous error; 2 warnings emitted
